src/Adapter/Module/Repository/ModuleRepository.php

```diff
@@ -24,72 +24,71 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  */
 
-namespace PrestaShopBundle\Kernel;
+declare(strict_types=1);
 
-use Doctrine\DBAL\Connection;
+namespace PrestaShop\PrestaShop\Adapter\Module\Repository;
+
+use Module;
+use PrestaShop\PrestaShop\Adapter\AbstractObjectModelRepository;
+use PrestaShop\PrestaShop\Core\Domain\Module\Exception\ModuleNotFoundException;
+use PrestaShop\PrestaShop\Core\Domain\Module\ValueObject\ModuleId;
+use PrestaShop\PrestaShop\Core\Exception\CoreException;
 use Symfony\Component\Finder\Finder;
 
 /**
- * Before booting the PrestaShop application in Symfony context,
- * we register every installed modules.
+ * Methods to access data source of Module
  */
-final class ModuleRepository
+class ModuleRepository extends AbstractObjectModelRepository
```

Comment:
BC Break as the class is not anymore final

Reply:
Bc breaks as the class is removed and replaced by another one :)

Reply:
This class is a new one

Reply:
It was when the class was move from the Kernel directory.